package.json

```diff
@@ -46,6 +46,7 @@
     "react": "^0.14.7 || ^15.0.0"
   },
   "dependencies": {
+    "babel-plugin-react-docgen": "^1.4.1",
```

Comment:
Why do we need this and the above entry in the `.babelrc`?
I don't think we need it.

React Storybook do all of this for you.